packages/web-auth0-auth-client/src/WebAuth0AuthClient.ts

```diff
@@ -33,17 +32,17 @@ const isEmptyOrNil = R.either(
 );
 
 const isEmailVerified = R.pipe(
-  R.path(['idTokenPayload', 'email_verified']),
+  R.pathOr(undefined, ['idTokenPayload', 'email_verified']),
```

Comment:
What's wrong with plain `R.path`?

Reply:
It does not return undefined if passed null instead of an object. I mean typing for R.path.